blocks/hello-there/hello-there.css

```diff
@@ -23,17 +23,21 @@
 
 @media (min-width: 1024px) {
   .hello-there {
-    padding-top: 84px;
-    margin: 0 40px;
+    padding: 84px 40px;
     column-gap: 40px;
     grid-template-rows: auto 257px auto;
   }
 }
 
 @media (min-width: 1280px) {
   .hello-there {
-    padding-top: 100px;
-    margin: 0 40px;
+    padding: 100px 40px 0;
     grid-template-rows: auto 307px auto;
   }
 }
+
+@media (min-width: 1400px) {
+  .hello-there {
+    padding: 100px 120px 0;
```

Comment:
If you set screen size for 1350px for example, the "hello-there" part becomes inconsistent with the rest of the page.  On second thought it probably because of the (min-width: 1400px), but if you fix it the inconsistency will just occur earlier.

Reply:
The hello-there part is like that because, the rest of the page has a max-width.